models/channel.go

```diff
@@ -3,7 +3,7 @@ package models
 import "github.com/google/uuid"
 
 type Channel struct {
-	ChannelID   string
+	ChannelID   UserID
```

Comment:
Should this be `ChannelId` defined in the same manner as `UserId`?  If they will always be the same (ie. string, uuid, etc) then maybe just a `ModelId` or `Id` type?

Reply:
ChannelID and UserID are both Twitch user ids
Should they be separate types, or will renaming to just ID make more sense?